scripts/workers/remote_submission_worker.py

```diff
@@ -25,11 +25,15 @@
 
 logger = logging.getLogger(__name__)
 
+logging.basicConfig(level=logging.INFO)
+
 AUTH_TOKEN = os.environ.get("AUTH_TOKEN")
 DJANGO_SERVER = os.environ.get("DJANGO_SERVER", "localhost")
 DJANGO_SERVER_PORT = os.environ.get("DJANGO_SERVER_PORT", "8000")
+DJANGO_SERVER_PROTOCOL = os.environ.get("DJANGO_SERVER_PROTOCOL", "http")
```

Comment:
@rcannood instead of taking protocol, port and server DNS explicitly we switched to passing the EvalAI endpoint as a single environment variable. So the value of endpoint would be `https://eval.ai` for prod and `http://localhost:8000` for dev environment. As we handle redirection of requests on our nginx reverse proxy we don't have to pass the port in the request url for production and staging servers.